include/gc/Dialect/Linalgx/LinalgxStructuredOps.td

```diff
@@ -104,4 +104,159 @@ def Linalgx_SigmoidOp : LinalgxStructuredBase_Op<"sigmoid",
   }];
 }
 
+def Linalgx_Mmt2DVnniOp 
+    : LinalgxStructuredBase_Op<"mmt2d_vnni", [AttrSizedOperandSegments]> {
+  let summary = "Transposed matmul with 2d input and vnni packed weights";
+  let description = [{
+    Supported format: A[M, N] * B[N0, K0, k, n, v] -> C[M, K], with:
+      N = N0 * n
+      K = K0 * k * v; v = (1, 2, 4)
```

Comment:
Please use M/K for A,  K/N for B, and M/N for C according to convention.
Why do we put `v=1` here?